### ISSUE TITLE
Alternative shows during drag even if showAlternatives: false

### ISSUE THREAD

**Issue by perliedman** [perliedman/leaflet-routing-machine#319]

**Comment by mdomnita** [comment]
Happens to me sometimes, sometimes alternative remains open after I release the mouse button and I can even switch to it by clicking with the mouse.

**Comment by mateusbd** [comment]
After routing, zoom in. The alternate route loads. Maybe checking showAlternatives at the start of the function _onZoomEnd (control.js) resolves this.

**Comment by yeganehaym** [comment]
yeah
it happened to me when I was zooming on the map
I'm not familiar with this js file but solved it by this code to _updateLineCallback function

```
	_updateLineCallback: function(err, routes) {
			if (!err) {
				routes = routes.slice();
				var selected = routes.splice(this._selectedRoute.routesIndex, 1)[0];
				if(!this.options.showAlternatives)
					{
						routes=[];
					}
				this._updateLines({route: selected, alternatives: routes });
			} else if (err.type !== 'abort') {
				this._clearLines();
			}
		},
```
this code is new
```
if(!this.options.showAlternatives)
					{
						routes=[];
					}
```

**Comment by Brandhor** [comment]
since #458 has been closed I'll post it here as well https://jsfiddle.net/4j7050te/1/

**Comment by perliedman** [comment]
@Brandhor yes - thanks for the example!

This issue is closed as well, since I have fixed the bug. The fix is included in the newly released version 3.2.8.

**Comment by Brandhor** [comment]
thanks, I didn't notice you closed it already
